Similarity_Between_Sherlock_stories.py

```diff
@@ -0,0 +1,336 @@
+"""
+This program calculates the cosine similarity of all Sherlock Holmes books written
+by Sir. Arthur Connan Doyle and the first book in the series, A Study In Scarlet.
+It graphs the cosine similarity against time to investiage the evolution of
+Doyle's writing.
+
+
+Author: MJ-McMillen
+"""
+from __future__ import division
+import requests #used to get books from guttenberg
+import math
+#used to plot
+import numpy as np
+import matplotlib.pyplot as plt
+import wikipedia
+
+
+#A_Study_In_Scarlet = requests.get('http://www.gutenberg.org/ebooks/244.txt.utf-8').text
+A_Study_In_Scarlet= open('astudyinscarlet.txt.utf-8')
+A_Study_In_Scarlet_lines = A_Study_In_Scarlet.readlines()
+full_Study = A_Study_In_Scarlet_lines[63:4713]
+
+#A_Sign_Of_Four = requests.get('http://www.gutenberg.org/ebooks/2097.txt.utf-8').text
+A_Sign_Of_Four= open('A_Sign_Of_Four.txt.utf-8')
+A_Sign_Of_Four_lines = A_Sign_Of_Four.readlines()
+full_Sugn = A_Sign_Of_Four_lines[34:4576]
+
+#Box = requests.get('http://www.gutenberg.org/ebooks/2344.txt.utf-8').text
+Box= open('cardboardbox.txt.utf-8')
+Box_lines = Box.readlines()
+full_Box = Box_lines[35:905]
+
+#Bow = requests.get('http://www.gutenberg.org/ebooks/2350.txt.utf-8').text
+Bow= open('last_bow.txt.utf-8')
+Bow_lines = Bow.readlines()
+full_bow = Bow_lines[36:776]
+
+#Lodge = requests.get('http://www.gutenberg.org/ebooks/2343.txt.utf-8').text
+Lodge= open('wisterialodge.txt.utf-8')
+Lodge_lines = Lodge.readlines()
+full_Lodge = Lodge_lines[35:1329]
+
+#Hound = requests.get('http://www.gutenberg.org/ebooks/2852-0.txt.utf-8').text
+Hound= open('The_Hound.txt.utf-8')
+Hound_lines = Hound.readlines()
+full_Hound = Hound_lines[32:6859]
+
+#The_valley = requests.get('http://www.gutenberg.org/ebooks/3289.txt.utf-8').text
+The_valley= open('the_valley_of_fear.txt.utf-8')
+The_valley_lines = The_valley.readlines()
+full_valley = The_valley_lines[32:6801]
+
+#The_Dying = requests.get('http://www.gutenberg.org/ebooks/2347.txt.utf-8').text
+The_Dying= open('The_Dying_Detective.txt.utf-8')
+The_Dying_lines = The_Dying.readlines()
+full_Dying = The_valley_lines[36:778]
+
+#Memoirs_get = requests.get('http://www.gutenberg.org/ebooks/834.txt.utf-8').text
+Memoirs_get= open('memoirs_of_sherlock_holmes.txt.utf-8')
+full_memoir =  Memoirs_get.readlines()
+AMT = ["The Silver Blaze","The Yellow Face",
```

Comment:
This variable name and few variables that follow this, things like `ATM` and `AMB`, are not a straightforward name. A comment describing what these variables are would be helpful.